pom.xml

```diff
@@ -6,7 +6,7 @@
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
     <!-- Baseline Jenkins version you use to build and test the plugin. Users must have this version or newer to run. -->
-    <version>2.17</version>
+    <version>2.33</version>
```

Comment:
@fcojfernandez Why not the latest one?

Reply:
Version 2.33 was latest version at the time of the Pull Request. At present time, latest version is 2.36. I've upgraded and tested.

@recena @andresrc what do you think about this upgrade now?